cdk/src/open-data-platform/app-plane/tileserver/tileserver.ts

```diff
@@ -35,14 +35,16 @@ export class TileServer extends Construct {
       cpu: 512, // measured in milliCPU; or 0.5 vCPU.
       taskImageOptions: {
         // Source: https://github.com/urbica/martin
-        image: ecs.ContainerImage.fromRegistry('urbica/martin'),
+        // TODO: Rename this reposity or tag to describe that it's the tile server, not the platform.
+        image: ecs.ContainerImage.fromRegistry('public.ecr.aws/m2w9u0k4/open-data-platform:latest'),
```

Comment:
is this just where the container is hosted on AWS?

Reply:
Yeah, I had to manually upload it since it didn't exist in any of the public AWS repositories yet.